client/signup/jetpack-connect/install-step.jsx

```diff
@@ -15,83 +16,100 @@ import JetpackExampleConnect from './example-components/jetpack-connect';
 
 const NEW_INSTRUCTIONS_JETPACK_VERSION = '4.2.0';
 
-export default React.createClass( {
-	displayName: 'JetpackInstallStep',
+class JetpackInstallStep extends Component {
 
```

Comment:
👍  a2b9581e2

Eventually I'll internalize this and you won't have to correct me on every PR 🙂